### ISSUE TITLE
ArchLinux advisories

### ISSUE THREAD

**Issue by noraj** [google/osv.dev#2733]
Any plan to add support for ArchLinux?

- **AVG** : ArchLinux Vulnerability Group - https://security.archlinux.org/
- **ASA** : ArchLinux Security Advisory - https://security.archlinux.org/advisory

**Comment by oliverchang** [comment]
Thanks for filing this issue. We typically depend on the upstream sources to adopt the OSV schema so that we can ingest it into our DB. 

@andrewpollock @hogo6002 should we try reaching out to Arch Linux here?

**Comment by noraj** [comment]
I don't know what public sources are available outside [this RSS feed](https://security.archlinux.org/advisory/feed.atom).

If you need to contact the AL security team, [here](https://wiki.archlinux.org/title/Arch_Security_Team#Contribute) are some pointers.

**Comment by andrewpollock** [comment]
https://wiki.archlinux.org/title/Arch_Security_Team#Team_members looks like the individuals to try and make initial contact with.

I'll drop the ones I can identify email addresses for a quick note pointing them at this issue.

**Comment by noraj** [comment]
@andrewpollock Did you get some answers from the AL team?

**Comment by andrewpollock** [comment]
@noraj no, I haven't heard back from anyone that I reached out to over email from https://wiki.archlinux.org/title/Arch_Security_Team#Team_members

**Comment by andrewpollock** [comment]
We eventually got a response from them, and they seem to be under-resourced to do the engineering work on their end to do what's necessary to make the data available. Discussions are ongoing, though.

**Comment by noraj** [comment]
Stale bots are really cancer IMO.
